assignments/arrays.js

```diff
@@ -63,32 +63,57 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+for(let i = 0; i < inventory.length ; i++){
+    if (i === 33){
+        console.log('Car 33 is a ' + inventory[32]['car_year'] + ' ' + inventory[32]['car_make'] + ' ' + inventory[32]['car_model'] );
```

Comment:
```suggestion
        console.log('Car 33 is a ' + inventory[i]['car_year'] + ' ' + inventory[i]['car_make'] + ' ' + inventory[i]['car_model'] );
```
So you are fixing what's happening above by specifying the index you want, but lets not do that. You made the loop, lets make use of it. The way we've modified the if statement above, we know that i is going to be the index we want, where the id is 33. So we can access the index we want by just using i, instead of specifying it directly.